Userland/Libraries/LibWeb/HTML/HTMLInputElement.h

```diff
@@ -198,16 +196,17 @@ class HTMLInputElement final
     bool value_as_number_applies() const;
     bool step_applies() const;
     bool step_up_or_down_applies() const;
+    bool select_applies() const;
     bool selection_or_range_applies() const;
 
+    virtual bool is_child_node_selectable(DOM::Node const&) const override;
+
     WebIDL::ExceptionOr<void> set_selection_start_for_bindings(Optional<WebIDL::UnsignedLong> const&);
```

Comment:
(If we do need to keep the semantics change on which element overrides the bindings, these declarations should be removed)

Reply:
I'll resolve this conversation for now, pending the outcome of the other conversations :)

Reply:
What I meant here was that these methods (on the HEAD of your branch) are currently unimplemented / unreferenced and should be removed:
* `set_selection_start_for_bindings` / `selection_start_for_bindings`
* `set_selection_end_for_bindings` / `selection_end_for_bindings`

Reply:
Ooh that's a silly leftover from a bad conflict resolution. Thanks!